i18n/en.json

```diff
@@ -84,5 +84,7 @@
     "usage-consent-header": "Send usage reports",
     "usage-consent-explanation": "Help make the app better by letting us know how you use it. Data collected is anonymous. Read privacy policy for more details.",
     "usage-consent-yes": "Yes",
-    "usage-consent-no": "No"
+    "usage-consent-no": "No",
+    "usage-consent-prompt": "Help make the app better by letting us know how you use it. You can change your preference at anytime from the settings menu. Data collected is anonymous.",
```

Comment:
Not sure why this message exists and is different than `usage-consent-explanation`. It looks like it's the same question being asked in 2 different places.

Reply:
Will check with design